lib/count_cdn_logs.rb

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
```

Comment:
Ruby 2.0 is utf-8 by default, this isn't neccessary anymore if we add the appropriate `.ruby-version`